docs/index.rst

```diff
@@ -205,6 +205,70 @@ Topic Management
       .. figure:: images/create_topic.png
          :alt: image
 
+7. When Confluent Monitoring Interceptors are configured on any Kafka-based clients, they write metadata to a topic called ``_confluent-monitoring``. Kafka-based clients include producers, consumers, Connect, KSQL, etc. 
+   |c3| reads that topic to do expert stream monitoring for message delivery assurance and performance for throughput and latency.
+   From that topic, you can also verify which producers are writing to which topics and which consumers are reading from which topics.
+   Checkout the sample script provided into the repo (Note: this is for demo purposes only, not suitable for production).
```

Comment:
Consider saying "An example script is provided with the repo [...]".

Reply:
Ack